<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QComboBox" name="cmbRegion">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="txtMonth">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::MonthSection</enum>
            </property>
            <property name="displayFormat">
             <string>M</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cmbCutOff">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Year</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Region</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="btnGetSCNames">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Get SC List</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Month</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="btnGetData">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Get DAT File</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="txtYear">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Cut-Off</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QProgressBar" name="progressBar">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>List of Uploaded SC</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mysql Data Fetcher</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <column>
           <property name="text">
            <string>SC Names</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblStatus">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
